### ISSUE TITLE
Not working on Audi A4

### ISSUE THREAD

**Issue by bhargav10** [thekirankumar/carstream-android-auto#141]
Tried unblocking app but didn't work. Can't see CarCast app when I click exit option in Android auto.

**Comment by shmykelsa** [comment]
Have you uninstalled CarStream before using the patcher?

You may want to retry

**Comment by bhargav10** [comment]
Yes followed same steps as shown in video. Uninstalled CarStream app and the installed new patch build.

**Comment by shmykelsa** [comment]
After running the patcher you don't have to do nothing else.

**Comment by bhargav10** [comment]
It was suggested in video to uninstall old app and Install new patched app. However I see CarStream in list of music app but not listed when I exit Android auto. It shows return to my car menu!

**Comment by shmykelsa** [comment]
Run the guided windows program instead. Uninstall CarStream, and then run the patcher. After, plug your phone to car

**Comment by bhargav10** [comment]
I tried all possible build options. Looks like on exit "Return to Audi" option is overriding CarStream option. Can you please fix this?

**Comment by bhargav10** [comment]
I tried all possible build options. Looks like on exit "Return to Audi" option is overriding CarStream option. Can you please fix this?

**Comment by shmykelsa** [comment]
Have you actually installed CarStream with the APK after fixing it on the PC?

**Comment by wrecklesswun** [comment]
Same on S4 2018
